tests/Integration/Behaviour/Features/Context/Domain/Discount/DiscountFeatureContext.php

```diff
@@ -188,6 +189,60 @@ public function createDiscount(string $discountReference, string $discountType,
         }
     }
 
+    /**
+     * @When I update discount :discountType with the following properties:
+     */
+    public function editDiscount(string $discountReference, TableNode $node): void
+    {
+        $data = $this->localizeByRows($node);
+        $discountId = $this->getSharedStorage()->get($discountReference);
+        $command = new UpdateDiscountCommand($discountId);
+
+        if (isset($data['name'])) {
+            $command->setLocalizedNames($data['name']);
+        }
+        if (isset($data['highlight'])) {
+            $command->setHighlightInCart(PrimitiveUtils::castStringBooleanIntoBoolean($data['highlight']));
+        }
+        if (isset($data['allow_partial_use'])) {
+            $command->setAllowPartialUse(PrimitiveUtils::castStringBooleanIntoBoolean($data['allow_partial_use']));
+        }
+        if (isset($data['priority'])) {
+            $command->setPriority((int) $data['priority']);
+        }
+        if (isset($data['active'])) {
+            $command->setActive(PrimitiveUtils::castStringBooleanIntoBoolean($data['active']));
+        }
+        if (isset($data['valid_from'])) {
```

Comment:
Same suggestion as above in the command, editing only one of the two fields is a legit use case IMHO